dwl.c

```diff
@@ -1808,9 +1810,6 @@ outputmgrapplyortest(struct wlr_output_configuration_v1 *config, int test)
 	else
 		wlr_output_configuration_v1_send_failed(config);
 	wlr_output_configuration_v1_destroy(config);
-
-	/* TODO: use a wrapper function? */
-	updatemons(NULL, NULL);
```

Comment:
Weird.  Do you remember what kind of changes didn't apply, and are we sure it's not a wlroots bug?  These lines came from cb01ce9, in case that helps jog your memory.

(I don't have an external on me right now, so I had to test using the wl backend instead of drm.)